website/docs/faqs/Git/github-permissions.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "I'm seeing a 'GitHub and dbt Cloud latest permissions' error"
+description: "GitHub and dbt Cloud permissions error"
+sidebar_label: "GitHub and dbt Cloud permissions error"
+---
+
+If you're seeing a `This account needs to accept the latest permissions for the dbt Cloud GitHub App` error in the dbt Cloud IDE &mdash; this usually occurs when the permissions for the dbt Cloud GitHub App are out of date.
+
+To solve this issue, you'll need to update the permissions for the dbt Cloud GitHub App in your GitHub account. Here's a couple of ways you can do it:
+
+#### First workaround
+
+You'll need to reach out to your Github organization admin to update the permissions for the dbt Cloud GitHub App.
+
+1. Go directly to GitHub to determine if any updated permissions are required.
+2. In GitHub, go to your organization **Settings** (or personal if using a non-organization account).
+3. Then navigate to **Applications** to identify any necessary permission changes.
+4. For more info on GitHub permissions, refer to [access permissions](https://docs.github.com/en/get-started/learning-about-github/access-permissions-on-github).
+
+#### Second workaround
+
+Disconnect the GitHub and dbt Cloud integration in dbt Cloud.
+
+1. In dbt Cloud, go to **Account Settings**.
+2. In **Projects**, select the project that is experiencing the issue.
+3. Click the repository link under **Repository**.
+4. In the **Repository details** page, click **Edit**.
+5. Click **Disconnect** to remove the GitHub integration.
```

Comment:
Should the leave it disconnected or is the implication that they'll re-connect?  What happens if they need that connection?

Reply:
added some clarity here good shout